### PR TITLE
Upgrade Kubernetes 1.7 release channel to 1.7.9

### PR DESCRIPTION
**Pull request by jackfrancis**
**Release note**:

```
Upgrade Kubernetes 1.7 release channel to 1.7.9
```

### CODE REVIEW

File: pkg/acsengine/const.go [hunk 1]
```diff
@@ -123,7 +123,7 @@ var KubeConfigs = map[string]map[string]string{
 		"gclowthreshold":  strconv.Itoa(DefaultKubernetesGCLowThreshold),
 	},
 	api.KubernetesVersion1Dot7Dot7: {
-		"hyperkube":       "hyperkube-amd64:v1.7.7",
+		"hyperkube":       "hyperkube-amd64:v1.7.8",
```
Comment (anhowe) [inline pkg/acsengine/const.go]: zip file update needed.

Comment (jackfrancis) [reply pkg/acsengine/const.go]: @JiangtianLi please let us know when we have a `v1.7.8` windows artifact

**Comment by lachie83** [comment]
1.7.9 just dropped :)

**Comment by JiangtianLi** [comment]
are we still making 1.7.8?

**Comment by jackfrancis** [comment]
@JiangtianLi let's skip to 1.7.9, I'll update this PR accordingly!

**Comment by JiangtianLi** [comment]
1.7.9 doesn't have a tag yet. so our build script doesn't automatically work. have to manually build...

**Comment by JiangtianLi** [comment]
1.7.9 tag was added 7 min ago right after my previous comment. How responsive we are!

**Comment by JiangtianLi** [comment]
@jackfrancis  uploaded zip and tested. please change to v1.7.9-1intwinnat.zip accordingly.

**Pull request merged**